apps/site/pages/en/learn/typescript/introduction.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introduction to TypeScript
+layout: learn
+authors: sbielenica, ovflowd, vaishnav-mk, AugustinMauroy
+---
+
+# Introduction to TypeScript
+
+## What is TypeScript
+
+**[TypeScript](https://www.typescriptlang.org)** is an open-source language maintained and developed by Microsoft. It's loved and used by a lot of software developers around the world.
+
+Basically, it's a superset of JavaScript that adds new capabilities to the language. The most notable addition is static type definitions, something that is not present in plain JavaScript. Thanks to types, it's possible, for example, to declare what kind of arguments we are expecting and what is returned exactly in our functions or what's the exact shape of the object that we are creating. TypeScript is a really powerful tool and opens a new world of possibilities in JavaScript projects. It makes our code more secure and robust by preventing many bugs before the code is even shipped - it catches problems during code development and integrates wonderfully with code editors like Visual Studio Code.
```

Comment:
I don't think "superset" is actually correct, IIUC it's _almost_ a superset with a few exceptions (e.g. ``console.log(`I'm ${(0).toFixed.apply<Number, String>(1) ? 'Type' : 'Java'}Script`)`` outputs something different if it's interpreted as TS or JS). Maybe we can say it's a language "very close to JavaScript, with a few additions"

Reply:
I think that for the majority of users (who are here to discover, I suppose) it makes no difference. And the term “superset” makes sense in this sense, because it puts a layer on top of the js.

Reply:
I agree with @aduh95 "superset" is not the correct term, also is an outdated TypeScript definition, we should go with the official Microsoft definition for TypeScript:

> TypeScript is a strongly typed programming language that builds on JavaScript, giving you better tooling at any scale.

Here is a more complete definition taken from the official [site](https://www.typescriptlang.org/):

>What is TypeScript?
JavaScript and More
TypeScript adds additional syntax to JavaScript to support a tighter integration with your editor. Catch errors early in your editor.

>A Result You Can Trust
TypeScript code converts to JavaScript, which runs anywhere JavaScript runs: In a browser, on Node.js or Deno and in your apps.

>Safety at Scale
TypeScript understands JavaScript and uses type inference to give you great tooling without additional code.

When possible, definition must be the official one. Also I would avoid to mention a particular code editor.